docs/usage/distrobox-assemble.md

```diff
@@ -214,3 +220,15 @@ each option corresponds to one of the `create` flags.
 	[ubuntu]
 	image=ubuntu:latest
 	pre_init_hooks="export SHELL=/bin/bash;"
+
+**Include example**
```

Comment:
```suggestion
**Include example (inherit fields from another distrobox)**
```

Reply:
Yes, I can see where you're coming from and for me it's better than inherit as a keyword. I just wanted a less concise title, sorry for being pedantic!